edk2toolext/versioninfo/versioninfo_helper.py

```diff
@@ -0,0 +1,556 @@
+## @file versioninfo_helper.py
+# This module contains helper functions for creating VERSIONINFO resources
+# from json files, along with the functions to output version information from
+# PE/PE+ files.
+#
+##
+# Copyright (C) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import pefile
+import sys
```

Comment:
not a fan of importing sys into a non cli module.  Looks like logging could remove your dependency on sys.